rdr_service/genomic/genomic_job_controller.py

```diff
@@ -1968,22 +1968,35 @@ def check_gcr_14day_escalation(self):
         escalate_pids = [pid[0] for pid in escalate_participants]
         if notification_email_addresses and escalate_pids:
             notified_participants = []
-            for pid in escalate_participants:
-                EmailService.send_email(
-                    Email(
-                        recipients=notification_email_addresses,
-                        subject='GCR Outreach 14 Day Escalation',
-                        plain_text_content=str(pid)
+            error_pids = []
+            for pid in escalate_pids:
+                try:
+                    EmailService.send_email(
+                        Email(
+                            recipients=notification_email_addresses,
+                            subject='GCR Outreach 14 Day Escalation',
+                            plain_text_content=str(pid)
+                        )
                     )
-                )
-                notified_participants.append({
-                    'participant_id': pid,
-                    'created': clock.CLOCK.now(),
-                    'modified': clock.CLOCK.now()
-                })
+                # pylint: disable=broad-except
+                except Exception as e:
+                    error_pids.append(pid)
+                    logging.error(f'Failed to send GCR Outreach Escalation email for {pid}: {e}')
+                else:
+                    notified_participants.append({
+                        'participant_id': pid,
+                        'created': clock.CLOCK.now(),
+                        'modified': clock.CLOCK.now()
+                    })
 
             notified_dao.insert_bulk(notified_participants)
-            self.job_result = GenomicSubProcessResult.SUCCESS
+            if error_pids:
+                self.genomic_alert_slack.send_message_to_webhook(
+                    message_data={'text': f"GCR Outreach Escalation email had errors for PIDs: {error_pids}"}
+                )
+                self.job_result = GenomicSubProcessResult.ERROR
+            else:
+                self.job_result = GenomicSubProcessResult.SUCCESS
```

Comment:
I think a different approach for this would be better. Since you are already writing the records and pids for this to a table maybe we can add a message_sent bool to the table to denote that the API request for Sendgrid has been successful and if it wasn't tag it with False, so that way we have a way to keep track of the pids that were not sent to Color on the escalation emails instead of keeping them in slack. 

Also I would edit the slack message to say that the Scheduled escalation email was unsuccessful with sending the payload of the pids since you already have them in db now.